lighthouse-core/gather/gatherers/seo/robots-txt.js

```diff
@@ -37,10 +18,10 @@ class RobotsTxt extends FRGatherer {
    * @return {Promise<LH.Artifacts['RobotsTxt']>}
    */
   snapshot(passContext) {
-    return passContext.driver.executionContext.evaluate(getRobotsTxtContent, {
-      args: [],
-      useIsolation: true,
-    });
+    const robotsUrl = new URL(passContext.url);
```

Comment:
Should we fork this to only use the fetcher on M92 and higher?

Reply:
I figured that switching to using the iframe fetcher would only increase the number of pages that fail this audit because of a CSP with `frame-src` restricted.